### ISSUE TITLE
About multiperson inference of 3DPW dataset

### ISSUE THREAD

**Issue by doublelei** [Arthur151/ROMP#97]
Hi 

Thanks for your wonderful work. I'm trying to save the vertices result inferenced by your model by running 
`python -m romp.test --configs_yml=configs/eval_3dpw_test_resnet.yml`

And I add 

`for i, name in enumerate(outputs['meta_data']['imgpath']):
         result[name] = {'vertices': outputs['verts'][i].detach().cpu().numpy()}`
to [eval](https://github.com/Arthur151/ROMP/blob/master/romp/eval.py#L55)

But I found the shape of vertices is 6890 x 3 instead of num_person x 6890 x 3. Why there is only one person in the inference result of each image? And how to got the multiperson  result for 3DPW dataset? Appreciate for your help in advance.

**Comment by Arthur151** [comment]
Please change your added code from 
```
for i, name in enumerate(outputs['meta_data']['imgpath']): 
      result[name] = {'vertices': outputs['verts'][i].detach().cpu().numpy()}
```
to 
```
for i, name in enumerate(outputs['meta_data']['imgpath']): 
      result[name]['vertices'].append(outputs['verts'][i].detach().cpu().numpy())
```
I think you can see my point.
Thanks for providing code for me to debug.
